stacks/nodejs-mongodb/devfile.yaml

```diff
@@ -0,0 +1,77 @@
+schemaVersion: 2.2.2
```

Comment:
should we use 2.3.0 ?

Reply:
@ibuziuk looks like 2.3.0 schema version is not supported by odo v3 yet
see this comment: https://github.com/devfile/registry/pull/602#pullrequestreview-2803938046